docs/Documentation/Scripting/Building-Blocks/Variables-List.md

```diff
@@ -104,10 +104,6 @@ variables:
 ```
 
 ## Other Variables
-```
-%condition.myCondition%
```

Comment:
Please move this to the relevant section instead of deleting it.

Reply:
It is already in the relevant section line 79

Reply:
This belongs in line 31.